pingouin/bayesian.py

```diff
@@ -10,6 +10,8 @@
 def _format_bf(bf, precision=3, trim='0'):
     """Format BF10 to floating point or scientific notation.
     """
+    if type(bf) == str:
```

Comment:
Why is this needed?

Reply:
`pairwise.pairwise_ttests` internally calls `parametric.ttest`, which also returns a post-processed DataFrame. At the end of `pairwise_ttests`, `postprocess_dataframe` is called again, but we don't want to attempt to format the BF again (which is by then already a string).